examples/react-tsx/src/components/Hello.tsx

```diff
@@ -0,0 +1,5 @@
+import * as React from "react";
+
+export interface HelloProps { compiler: string; framework: string; }
```

Comment:
you could also define the HelloProps as type here, as you are not using any extention by now. interface is also fine though.

Reply:
you don't need to export the HelloProps. In this moment, they are just used in this file.

Reply:
thx. done 79d87ad